html/themes/custom/common_design_subtheme/css/styles.css

```diff
@@ -69,3 +69,37 @@
   vertical-align: middle;
   background: transparent url("../components/rw-brand/rw-icons-sprite.svg") -84px -108px no-repeat;
 }
+
+.block-local-actions-block nav {
```

Comment:
This chunk of CSS was improved in #627 — the code here will revert it to something less than ideal.

Reply:
ok scratch that, it's the same commit as in #627 so the changes in the other PR will survive if this is left alone.